devices/common_patches/dma/dma2d_v2.yaml

```diff
@@ -0,0 +1,14 @@
+# "Chrom-Art Accelerator" 2D DMA engine
+# Version 2
+
+"DMA2D*":
+  "[FB]GOR,OOR":
+    _modify:
+      LO:
+        bitWidth: 15
```

Comment:
I'm looking at RM0433 where this seems to be 16 bits?

Reply:
Yup, fixed now.